src/lib/selectors.ts

```diff
@@ -653,6 +655,32 @@ export const getResendFileMessage = (state: SendBirdState) => (
   )
 );
 
+export const getFetchChannelList = (state: SendBirdState) => (
+  (context: ChannelListProviderInterface) => {
+    const { premiumFeatureList = [] } = state?.stores?.sdkStore?.sdk?.appInfo ?? {};
+    const {
+      logger,
+      markAsDeliveredScheduler,
+      disableMarkAsDelivered,
+    } = state.config;
+    const {
+      channelSource,
+      channelListDispatcher,
+    } = context;
+
+    useFetchChannelList({
```

Comment:
It looks a bit awkward to call hook from the normal js function since it's not neither React component nor custom hook. https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks

Reply:
It will be called inside `React component `, but yeah, agree -> it customer would put this inside loop/conditional or something and potentially could break the rule of hook with this ~

Reply:
@HoonBaek , probably better to export the hook through `sendbird/uikit-react/Channel/context`

Reply:
Me and @HoonBaek  discussed, and we wont be exporting this as `sendbirdSelector`
We will add it to to the `ChannelListContext` and customer can use it through there


BTW ~ 
@HoonBaek When you have time, lets add documentation PR ~

Reply:
Yeah I just saw Hoon's message in Slack too. Like the idea 👍

Reply:
applied it~!